### PR TITLE
Marketingupdate

### PR DESCRIPTION
**Pull request by ladyleet**

### CODE REVIEW

File: app/components/mentor-card.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Comment (taras) [inline app/components/mentor-card.js]: You don't need to create a js file if you're not making any changes to the component's JS. Pls, remove this file

File: app/components/nav-bar.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Comment (taras) [inline app/components/nav-bar.js]: Pls, remove this file

File: app/components/site-footer.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Comment (taras) [inline app/components/site-footer.js]: Pls, remove this file

**Comment by taras** [comment]
Looks good, other than the few unnecessary files. Pls, remove those and I'll merge this

File: app/components/mentor-card.js (hunk 1 above)
Comment (ladyleet) [inline app/components/mentor-card.js]: done

File: app/components/nav-bar.js (hunk 2 above)
Comment (ladyleet) [inline app/components/nav-bar.js]: done

File: app/components/site-footer.js (hunk 3 above)
Comment (ladyleet) [inline app/components/site-footer.js]: done

**Comment by ladyleet** [comment]
@taras r?

**Comment by ladyleet** [comment]
- updated the site footer with social links.
- updated the about page.
- installed coin clicks add-on for my own entertainment while updating site. will uninstall when it's ready to go live. :P

**Comment by taras** [comment]
:+1:

**Pull request merged**